staging/src/k8s.io/kubectl/pkg/cmd/delete/delete.go

```diff
@@ -436,3 +444,16 @@ func (o *DeleteOptions) PrintObj(info *resource.Info) {
 	// understandable output by default
 	fmt.Fprintf(o.Out, "%s \"%s\" %s\n", kindString, info.Name, operation)
 }
+
+func (o *DeleteOptions) confirmation(info *resource.Info) bool {
+	fmt.Fprintf(o.IOStreams.Out, "Are you sure you want to delete %s/%s? (y/n): ", strings.ToLower(info.Mapping.GroupVersionKind.Kind), info.Name)
```

Comment:
How easy will it be to localize this into other languages if we need to?

Reply:
I believe it can be wrapped in `i18n.T()` like this, without an issue.  As long as the translation leaves the `%s`s in place.
```
i18n.T("Are you sure you want to delete %s/%s? (y/n): ")
```

However it gets a little tricky in this situation because `y/n` may not make sense for a given language, say French...

`Etes-vous sûr que vous voulez supprimer %s/%s (o/n): `

Well, the code would only react to `y`, so if the localization changed the `y/n` part of the prompt, it wouldn't work out well.

Is `y/n` pretty much universally understood across languages, even without having to know English well?  If so, then maybe it could be handled something like this, to prevent the "y/n" part from being able to be changed during translation:

```
i18n.T("Are you sure you want to delete %s/%s?") + " (y/n): "
```

Reply:
I changed with what Brian suggested ^